examples/autoscaler-webhook/main.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	v1alpha1 "agones.dev/agones/pkg/apis/stable/v1alpha1"
+	"agones.dev/agones/pkg/util/runtime" // for the logger
+)
+
+// Constants which define thresholds to trigger scalling up and scale factor
+const (
+	replicaUpperThreshold = 0.7
```

Comment:
We should create an issue to make those as environment variable flags, there is an opportunity for this to be used by anyone and not just as an example.